src/utils/helpers/notifications.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/prefer-default-export */
+import { sequelize } from '../../database/models';
+import { getUserRole } from '../../services/rolesService';
+
+const getNotifications = async (userId) => {
+  try {
+    const userRole = await getUserRole(userId);
+    if (userRole === 'Requester') {
+      const query =
+        'SELECT id,title,message,"tripId","createdAt" FROM notifications ' +
+        `WHERE "isRead"='false'` +
+        ` AND type='application'` +
+        ` AND (category ='status' OR category='comment')` +
+        ` AND type='application'` +
+        ` AND "addedBy"<>'${userId}'` +
+        ` AND "tripId" IN ` +
+        `(SELECT id FROM trips WHERE "userId"='${userId}')`;
+      const [results] = await sequelize.query(`${query}`);
+      return results;
+    }
+
+    if (userRole === 'Manager') {
+      const query =
+        'SELECT id,title,message,"tripId","createdAt" FROM notifications ' +
+        `WHERE "isRead"='false'` +
+        ` AND type='application'` +
```

Comment:
You should remove isRead selector as user may want also the notification read before